rules/do-you-always-rename-staging-url-on-azure/rule.md

```diff
@@ -23,38 +23,28 @@ If you use the default Azure staging web site Url, it can be difficult to rememb
 
 <!--endintro-->
 
-
-| Default Azure Url:
-<ul><li><strong style="line-height:20px;background-color:initial;"> <strong>sugarlearning<span style="color:#ff0066;">-staging</span>.azurewebsites.net</strong> </strong> 
-</li></ul> |
-| --- |
-
-
-
+::: greybox
+Default Azure Url:
+sugarlearning**-staging**\.azurewebsites.net
+:::
 ::: bad
-Figure: Bad example - Site using the default Url (hard to remember!!)  
+Figure: Bad example - Site using the default URL (hard to remember!!)  
 :::
 
-
-| Customized Url:
-<ul><li><strong style="line-height:20px;background-color:initial;"><font color="#ff0066">staging</font></strong> <span style="line-height:20px;background-color:initial;">.</span><strong style="line-height:20px;background-color:initial;">sugarlearning.com</strong> 
-</li></ul> |
-| --- |
-
-
-
+::: greybox
+Customized Url:
+**staging**\.sugarlearning.com
```

Comment:
Hi @tiagov8,

Please remove the `\` from the url

FROM
	**staging**\.sugarlearning.com

TO
	**staging**.sugarlearning.com